source/partials/_sponsor.html.haml

```diff
@@ -1,11 +1,11 @@
-%body.spons
+.sponsor
 	%h1.spons-head
 		Sponsor
 	
-	%ul.spons
-		%li
-			%a{:href => "https://flawlessinbound.ca/"}
-				%img.simg(src="/images/flawless-inbound-logo-full.png")
-	%h2.sponsus
-		%a{:href => "/"}
-			Sponsor Us Here!
\ No newline at end of file
+	.spons
+		%a{href: "https://flawlessinbound.ca/"}
+			%img.simg(src="/images/flawless-inbound-logo-full.png")
+	.sponsus
+		%h2.btn.btn-primary

```

Comment:
using btn style that's already been made, but
the button has a different hover action for hovering on the rectangle of the button vs on the letters
Not sure why that's happening?

Reply:
It may be because of the `h2` tag. Try removing that and see how it works?